alignak/basemodule.py

```diff
@@ -246,12 +245,14 @@ def stop_process(self):
         :return: None
         """
         if self.process:
-            logger.info("I'm stopping module %r (pid=%s)",
+            logger.info("I'm stopping module %r (pid=%d)",
                         self.get_name(), self.process.pid)
             self.process.terminate()
+            # Todo: get sure that 1 second is enough ...
```

Comment:
I'd rather let the timeout, and even increase it to 10 seconds, ... because if the process did not terminated after the timeout we force kill it ... then it sees better to let the child process some time to terminate properly before killing it.